src/Engine/TranskribusClient.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Engine;
+
+use App\Exception\OcrException;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TranskribusClient
+{
+
+    /** @var HttpClientInterface */
+    private $httpClient;
+
+    /** @var string transkribus access token. */
+    private $accessToken = '';
```

Comment:
No need for a default value here, as this will always be set in the constructor so will never actually have this value.